Spigot-Server-Patches/0613-Fix-curing-zombie-villager-discount-exploit.patch

```diff
@@ -0,0 +1,57 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: BillyGalbreath <blake.galbreath@gmail.com>
+Date: Tue, 8 Dec 2020 20:14:20 -0600
+Subject: [PATCH] Fix curing zombie villager discount exploit
+
+This fixes the exploit used to gain absurd trading discounts with infecting
+and curing a villager on repeat by simply resetting the reputation when it
+is cured.
+
+diff --git a/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java b/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java
+index 5596d9f42523dbcbad47d21a4b2fdba39e3c5590..d6809766d3ade10c1a48b31f6dcfdcaaa349c755 100644
+--- a/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java
++++ b/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java
+@@ -704,4 +704,9 @@ public class PaperWorldConfig {
+     private void fixClimbingBypassingCrammingRule() {
+         fixClimbingBypassingCrammingRule = getBoolean("fix-climbing-bypassing-cramming-rule", fixClimbingBypassingCrammingRule);
+     }
++
++    public boolean fixCuringZombieVillagerDiscountExploit = true;
++    private void fixCuringExploit() {
++        fixCuringZombieVillagerDiscountExploit = getBoolean("fix-curing-zombie-villager-discount-exploit", fixCuringZombieVillagerDiscountExploit);
++    }
+ }
+diff --git a/src/main/java/net/minecraft/server/EntityVillager.java b/src/main/java/net/minecraft/server/EntityVillager.java
+index 0388d1099f2a6d436a5a5e58bbbdae3ddab969e7..91ce75027a78fc33f43b3d56bb0cc99387ab8bdb 100644
+--- a/src/main/java/net/minecraft/server/EntityVillager.java
++++ b/src/main/java/net/minecraft/server/EntityVillager.java
+@@ -20,12 +20,9 @@ import javax.annotation.Nullable;
+ import org.apache.logging.log4j.Logger;
+ // CraftBukkit start
+ import org.bukkit.Bukkit;
+-import org.bukkit.craftbukkit.entity.CraftVillager;
```

Comment:
imports

Reply:
I need to figure out why IJ started doing this automatically recently. Sorry :3